index.html

```diff
@@ -1017,6 +1017,206 @@ <h3>Interface Definition</h3>
+            <dd>
+              <p>The amount of stream bytes sent on the QUIC connection, not including
+              retransmissions.</p>
+            </dd>
+            <dt><dfn><code>streamBytesReceived</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of stream bytes received on the QUIC connection, not including
+              retransmissions.</p>
+            </dd>
+            <dt><dfn><code>numOutgoingStreamsCreated</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long</a></span></dt>
+            <dd>
+              <p>The amount of outgoing QUIC streams created on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>numIncomingStreamsCreated</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long</a></span></dt>
+            <dd>
+              <p>The amount of incoming QUIC streams created on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>bytesReceived</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of total bytes received on the QUIC connection, including
+              duplicate data for streams.</p>
+            </dd>
+            <dt><dfn><code>packetsReceived</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of total bytes received on the QUIC connection, including
+              packets that were not processable.</p>
+            </dd>
+            <dt><dfn><code>packetsProcessed</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of total packets processed on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>bytesRetransmitted</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of bytes retransmitted on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>packetsRetransmitted</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of packets retransmitted on the QUIC connection.</p>
+            </dd>
+            <dt><dfn><code>packetsLost</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of packets abandoned as lost determined by the loss detection
+              algorithm.</p>
+            </dd>
+            <dt><dfn><code>packetsDropped</code></dfn> of type <span class=
+            "idlMemberType"><a>unsigned long long</a></span></dt>
+            <dd>
+              <p>The amount of packets dropped.</p>
```

Comment:
What's meant by "dropped" here?

Reply:
Ignored packets. In quiche this is with unrecognized packet headers, or connection IDs. Could change to packetsIgnored?

Reply:
Removed this.